### PR TITLE
readme update explaining new module creation

### PR DESCRIPTION
**Pull request by Logicon211**

### CODE REVIEW

**Comment by Logicon211** [comment]
I want to add some python docs before this gets merged

File: README.md [hunk 1]
```diff
@@ -59,6 +59,18 @@ Returns:
 We are thrilled you are considering contributing!
 Please read our [guidelines for contributing](CONTRIBUTING.md).
 
+## Creating new translator modules
```
Comment (delliott90) [inline README.md]: I'm thinking the Contributing bit should remain right above the Licensing bit.

Comment (Logicon211) [reply README.md]: Sure

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Requires Python 3.6
 
 Call the stix_shifter in the format of
 
-python stix_shifter.py `<data source>` `<input format>` `<STIX pattern>`
+python stix_shifter.py `<translator_module>` `<query or result>` `<data>`
```
Comment (mdazam1942) [inline README.md]: Ben's command line parser have some more changes in this section. Would that create any conflict?

Comment (Logicon211) [reply README.md]: Most likely. He'll make the update in his PR though, as it stands with this commit this is how it will work

**Pull request merged**